src/Sulu/Bundle/MediaBundle/Media/Storage/LocalStorage.php

```diff
@@ -11,12 +11,13 @@
 
 namespace Sulu\Bundle\MediaBundle\Media\Storage;
 
-use stdClass;
+use League\Flysystem\FileExistsException;
+use League\Flysystem\FileNotFoundException;
+use League\Flysystem\Filesystem;
```

Comment:
would not use Flysystem in the LocalStorage there should be a FlySystemStorage which implements `ExternalStorageInterface` to get the external urls.